mava/components/training/losses.py

```diff
@@ -259,20 +315,21 @@ def policy_loss_fn(
                             policy_params, observations
                         )
 
-                    log_probs = network.log_prob(distribution_params, actions)
-                    entropy = network.entropy(distribution_params)
+                    log_probs = network.log_prob(distribution_params, actions) * mask
```

Comment:
Is it necessary to mask the log_probs f we mask the clipped_surrogate_pg_loss? Maybe it is necessary. Just confirm this.

Reply:
Not necessary